Containers/nextcloud/cron.sh

```diff
@@ -1,7 +1,11 @@
 #!/bin/bash
 set -eu
 
+php -f /var/www/html/cron.php &
+
 while true; do
-    php -f /var/www/html/cron.php &
     sleep 5m
+    php -f /var/www/html/cron.php &
 done
+
+wait $!
```

Comment:
@Zoey2936 I suppose this is not going to work?

Reply:
no

Reply:
Could you imagine a syntax that we could use here that would fix the problem? Possibly adding a wrapper script or so?